### DISCUSSION TITLE
Support for passthrough API, avatars and newer features

### DISCUSSION THREAD

**Discussion by Yeisonlop10** [pmndrs/xr#215] [General] [answered]
Thank you for this great tool and the hard work behind it. I would like to know if there is any plan to include support for some of the Quest (2, pro) features like, avatars SDK (not sure if there is any support for WebXR yet), passthrough api, eye tracking, etc. thank you

**Comment by bbohlender** [comment, marked as answer]
- passthrough is supported
- gaze input is supported

avatar SDK is not available for WebXR (yet?)

react-three/xr v6 supports a bunch of new WebXR features (Object detection), and support for more features is comming (anchors, layers, ...)
